player/stream-test/js/script.js

```diff
@@ -2,19 +2,33 @@ window.onload = function getURLParams() {
   var manifestParam = getParamsQueryString('manifest');
   var streamFormatParam = getParamsQueryString('format');
   if (manifestParam && streamFormatParam) {
-    document.getElementById('stream_format').value = streamFormatParam;
-    document.getElementById('manifest').value = manifestParam;
+    document.getElementsByName('stream-format').forEach(function (element) {
+      if (element.value === streamFormatParam) {
+        element.checked = true;
+      }
+    })
+    document.getElementById('manifest-input').value = manifestParam;
+
+    loadPlayerFromControls();
+  }
+  else {
+    setupPlayer('dash', config.source.dash);
   }
 };
 
-var manifest = document.getElementById('manifest');
-manifest.onkeyup = handleKeyPress;
+var initialTimestamp, bufferChart, bitrateChart;
```

Comment:
just fyi: const and let are widely supported in browsers now, so you can use them in the demos too.
[caniuse const](https://caniuse.com/#feat=const)
[caniuse let](https://caniuse.com/#feat=let)

Reply:
The compiler we used for production builds in the `demo-framework` did not support ES6 or above, just y-day we got a update to be able to use ES6